homework.js

```diff
@@ -3,24 +3,60 @@
 // assign your variable the value 6
 // console log your variable
 
+let fruit = 6;
+console.log(fruit)
 
 // write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
+function product(num1, num2) {
+  let product = num1 * num2
+  console.log(product)
+  return product
+}
 
+product(4, 689)
 // Problem 2
 // write a function called evens that takes an array of numbers, removes 
 // all the numbers that are odd and returns a new array with only the even numbers
 // remember that even numbers have a remainder of 0 when you divide them by 2
 // call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
-
+function evens (numbers) {
+  for (num of numbers) {
+    if (num % 2 === 0 ) {
+     numbers.pop(num)
```

Comment:
This will always take the last item off an array, which is not exactly what you want. You want to either splice the array or push the desired values to a new array.